internal/utils/testutil/examples.go

```diff
@@ -34,6 +34,11 @@ var (
 	// ExamplesPath represents the entire examples directory.
 	ExamplesPath = filepath.Join(RootPath, "examples")
 
+	// nolint:golint
```

Comment:
why nolint is needed here?

Reply:
the lint can to change CExample to Example, because is considering sintaxe error